src/fragments/start/getting-started/reactnative/setup.mdx

```diff
@@ -16,7 +16,7 @@ cd amplified_todo
 Install the necessary dependencies by running the following command:
 
 ```sh
-npm install aws-amplify @aws-amplify/react-native @aws-amplify/rtn-web-browser @react-native-community/netinfo @react-native-async-storage/async-storage
+npm install aws-amplify @aws-amplify/react-native @aws-amplify/rtn-web-browser @react-native-community/netinfo @react-native-async-storage/async-storage react-native-get-random-values
```

Comment:
Should we callout that rtn-web-browser is only needed for customers using Federated Auth / Social Signin?

See: https://ui.docs.amplify.aws/react-native/getting-started/installation#packages

Reply:
Yes we should

Reply:
We can do that in a separate PR